src/addons/__tests__/ReactComponentWithPureRenderMixin-test.js

```diff
@@ -50,20 +50,20 @@ describe('ReactComponentWithPureRenderMixin', function() {
       getInitialState: function() {
         return {
           cut: false,
-          slices: 1,
+          slices: 1
```

Comment:
I'll open another PR reversing this rule, and rebase this PR on top of it after it merges. That seems like the least amount of overall effort.